package.json

```diff
@@ -6,20 +6,24 @@
   "author": "StackStorm <support@stackstorm.com>",
   "description": "A simple helpful robot for your Company, with StackStorm Support",
   "dependencies": {
+    "botbuilder": "^3.16.0",
+    "coffee-register": "1.0.0",
+    "coffee-script": "1.12.7",
     "hubot": "^3.1.1",
+    "hubot-botframework": "^0.10.1",
     "hubot-diagnostics": "0.0.2",
-    "hubot-redis-brain": "0.0.4",
-    "hubot-scripts": "^2.17.2",
-    "hubot-help": "^0.2.2",
-    "hubot-stackstorm": "0.9.2",
     "hubot-flowdock": "^0.7.8",
-    "hubot-matteruser": "^5.2.0",
-    "hubot-spark": "^2.0.0",
-    "hubot-xmpp": "^0.2.5",
-    "hubot-hipchat": "git+https://github.com/StackStorm/hubot-hipchat#sonnyp-patch",
+    "hubot-help": "^0.2.2",
+    "hubot-hipchat": "git+https://github.com/StackStorm/hubot-hipchat.git#sonnyp-patch",
     "hubot-irc": "^0.4.0",
+    "hubot-matteruser": "^5.2.0",
+    "hubot-redis-brain": "0.0.4",
+    "hubot-rocketchat": "^2.0.0",
+    "hubot-scripts": "^2.17.2",
     "hubot-slack": "^4.5.5",
-    "hubot-rocketchat": "^2.0.0"
+    "hubot-spark": "^2.0.0",
+    "hubot-stackstorm": "git+https://github.com/StackStorm/hubot-stackstorm.git#add-msteams",
```

Comment:
TODO: needs changing to _real_ released version of `hubot-stackstorm` prior merging

Reply:
Done.